L/libjulia/build_tarballs.jl

```diff
@@ -1,5 +1,5 @@
 include("common.jl")
 jllversion=v"1.7.0"
```

Comment:
I forgot, but what was the decision for the jllversion? Should I increment it or leave it? Since this builds a JLL that in the end contains artifacts for multiple Julia versions, perhaps there is no reason to keep this related to the most recent Julia version anymore, and so we could use anything, such as 1.7.1 or 1.8 or... ?